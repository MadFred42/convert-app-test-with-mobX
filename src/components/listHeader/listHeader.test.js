import React from 'react';
import ListHeader from './';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Testing <ListHeader/>', () => {
    it('ListHeader props', () => {
        const header = shallow(<ListHeader/>);
        expect(toJson(header)).toMatchSnapshot();
    })
})