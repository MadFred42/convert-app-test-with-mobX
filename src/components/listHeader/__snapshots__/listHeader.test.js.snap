// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <ListHeader/> ListHeader props 1`] = `
<ul
  className="row"
>
  <li
    className="col list-group-item some"
  >
    <h5>
      Наименование валюты
    </h5>
  </li>
  <li
    className="col list-group-item"
  >
    <h5>
      Актуальный курс*
    </h5>
  </li>
  <li
    className="col list-group-item"
  >
    <h5>
      Вчерашний курс*
    </h5>
  </li>
  <li
    className="col list-group-item"
  >
    <h5>
      Разница
    </h5>
  </li>
  <li
    className="col list-group-item"
  >
    <h5>
      Код валюты
    </h5>
  </li>
</ul>
`;
